foss4gna_24/big-EO-viz.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>was this necessary? the original source is geojson which looks to be 4326 already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Not necessary, it is shown just in case the layers are not in 4326 by default. </p>